pull language from "branch language" to "master"

JhowDroid opened the pull request:
Adding language file for german translation
Adding language file for spanish translation

@davidsantos-br

JhowDroid committed:
German Translation - Deutsch Übersetzung

Language file for german translation

JhowDroid committed:
Spanish Translation - Traducción Española

Language file for spanish translation

JhowDroid commented:
Adding language file for spanish translation

@davidsantos-br

davidsantos-br commented:
I'm replacing my hard drive, but I think I'll be able to check this out tomorrow.

JhowDroid committed:
Spanish Translation - Traducción Española

Proofreading and corrections by "Anjo da Noite"

JhowDroid committed:
Portuguese Translation - Tradução Português-PT

Language file for portuguese pt-PT translation

JhowDroid committed:
German Translation - Deutsch Übersetzung

updating line 35 to:
save_format: "Speichern Format “save format”",

JhowDroid committed:
Spanish Translation - Traducción Española

removing wrong character at the beginning of each line.

JhowDroid committed:
Merge remote-tracking branch 'refs/remotes/davidsantos-br/master' into language

JhowDroid committed:
Adding 3 new strings to language files

Adding 3 new strings to language files:
language_menu, this_language, about_to_restart.
